NAMESPACE

```diff
@@ -106,8 +106,8 @@ export(plotDualResponses)
 export(plotGain)
 export(prob)
 export(probit)
-export(sampleSize)
-export(saveSample)
```

Comment:
please keep the original `camelCase` way of the function names (as mentioned before), as the whole package is like that and I don't want to change too much. thx